fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -125,9 +129,10 @@ def parse_line(line, hide_output)
             CredentialsManager::AccountManager.new(user: @user).invalid_credentials
             UI.error("Please run this tool again to apply the new password")
           end
-        elsif $1.include?("Redundant Binary Upload. There already exists a binary upload with build")
+        elsif $1.include?("Redundant Binary Upload. You've already uploaded a build with build")
           UI.error($1)
-          UI.error("You have to change the build number of your app to upload your ipa file")
+          /build number (?<build_number>'.*') for version number (?<version_number>'.*')./ =~ $1
+          UI.error("You need to upload a binary with a build number greater than #{build_number}. Or upload a binary with a version greater than #{version_number}.")
```

Comment:
This error message is not very clear. Couldn't this be combined?
```suggestion
          UI.error("You need to upload a binary with a build number greater than #{build_number} or a version greater than #{version_number}.")
```
Or isn't this correct then?

Reply:
I'm not too hung up on exact wording but do want to make sure we don't leave any confusion for the user.
This error happens because for a given version - CFBundleShortVersionString (https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-111349) the user has uploaded a binary with a build number - CFBundleVersion (https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102364).

Would it be worth pointing them to the docs at all to help people understand how the two strings work?